sonic-syseepromd/scripts/syseepromd

```diff
@@ -115,6 +122,59 @@ class DaemonSyseeprom(daemon_base.DaemonBase):
 
         return True
 
+    def post_values_to_state_db(self, key, values):
+        try: 
+            state_db = daemon_base.db_connect(STATE_DB)
+            for k,v in values.items():
+                state_db.hset(key, k, str(v))
+        except Exception as e:
+            self.log_error(f'writing {k}:{v} to db failed with {e}')
+
+    def post_cpu_stats_to_db(self):
+        # Get CPU utilization
+        cpu_stats = {}
+        # read cpus load in the last 5 minutes
+        _, load5, _ = psutil.getloadavg()
+        cpu_count = os.cpu_count()
+        if not cpu_count: cpu_count = 1
+        # divide load by number of cpus and calculate percentage
+        utilization = round((load5 * 100) / cpu_count)
+        if utilization not in range(0,100):
+            self.log_error(f"calculated utilization {utilization} out of range, "
+                          f"load5={load5} cpu_count={cpu_count}")
+            utilization = 'N/A'
+        cpu_stats["utilization"] = utilization
+
+        # Get free RAM
+        memory_usage = psutil.virtual_memory()
+        cpu_stats["free_ram"] = round((memory_usage.available / memory_usage.total) * 100)
+        
+        self.post_values_to_state_db(CPU_UTIL_ENTRY, cpu_stats)
+
+    def post_disk_stats_to_db(self):
+        hdd_stats = {}
+        hdd_stats['free_space'] = 100 - round(psutil.disk_usage('/').percent)
+
+        ssd = SsdUtil("/dev/sda")
```

Comment:
Not all vendors are using SsdUtil, you may check src/sonic-utilities/ssdutil/main.py function import_ssd_api. You can see that some vendors are using a platform specific ssd implementation. But, syseepromd is a common daemon for all vendors.